### ISSUE TITLE
Consider finding a better name

### ISSUE THREAD

**Issue by martinthomson** [chris-wood/draft-wood-cfrg-blind-signatures#70]
Yes, "blind" is a term of art, but it is used in an unnecessarily pejorative fashion.

**Comment by FredericJacobs** [comment]
Thanks @martinthomson. This was actually the first issue we filed on this repo (#1) and it was closed. But we of course want to keep that discussion open and would welcome input on that from the broader community.

The term being widely used in the academic literature makes it hard to move away from it without causing some confusion about what our scheme implements.

We also thought through about what it was referring to and got some interesting inputs from persons in the blind community such as:
>  I'd even argue that in the case of the examples you mention it's an entirely appropriate use of the word because it describes the same thing - a situation in which someone does not have access to all their sensory characteristics or is not able to use all their sensory characteristics.

We considered "unlinkable signatures" to emphasize that between signing and verification, the signature should not be linkable but it doesn't convey all the nuance we were hoping for.
